docs/snippets/add-a-currency-symbol.md

```diff
@@ -0,0 +1,40 @@
+# Add a currency and symbol
+
+> This is a **Developer level** doc. If you are unfamiliar with code and resolving potential conflicts, select a [WooExpert or Developer](https://woocommerce.com/customizations/) for assistance. We are unable to provide support for customizations under our [Support Policy](http://www.woocommerce.com/support-policy/).
```

Comment:
Sorry, further piece of feedback. Per Brent, we can and should drop these developer-level doc warnings.

Reply:
Removing this note based on p1692723749470709/1692723559.960259-slack-C0516GKR69W